wirecard/oxid-ee
# WDPD 182: Guaranteed Invoice by Wirecard Final Frontend Payment Method

### This PR

* Applies the following restrictions to whether or not the **Guaranteed Invoice by Wirecard** payment is displayed to the user:
  * No digital goods must be part of the shopping basket
  * The selected currency must be one of the allowed currencies
  * The shipping country must be one of the allowed shipping countries
  * The billing country must be one of the allowed billing countries
  * If the setting "Identical Billing/Shipping Address" is enabled, shipping and billing county must be the same
* Adds unit tests for the methods introduced in https://github.com/wirecard/oxid-ee/pull/115 to keep 90% code coverage

### How to Test

* Activate the payment method
* Try different settings for:
  * Allowed Currencies
  * Allowed Shipping Countries
  * Allowed Billing Countries
* Check if the restrictions noted above are applied correctly

### Jira Links

* WDPD-182: https://jira.parkside.at/browse/WDPD-182
